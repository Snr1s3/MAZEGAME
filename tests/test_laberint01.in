gAAAAABmL2-dRnbqzDJWjHQqZ99vJkWL_Dbr1PJkuY-lHhlMYH5nTQfAdkjcVRda4Ice7K_8100keLr-xFRAHWSsKmgGH-TFvg==