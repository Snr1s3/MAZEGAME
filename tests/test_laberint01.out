gAAAAABmL2-dchyny603_UNBHdftCbTwjaGazyFFHBVSM-eV_A9nQmW-Fb22zA66_U5tXjQ1y56nV2lStqr_DeRM7LNpnNl799qfqLGhkVQyUldJnOQKSBfi0cYI8EtY0c0zeBwZs1rQ_SSgIvPE_C_lZlSL9BT1A8TH2M6mMMB1KwjS-RZqWCqvw2CwnZGFY4yUjohZIdF_Osb59th_DNhmIYRrI65uPgeLmjkgI6Rq9YvdCIa_mj934PWFLpxoP6_o42ORxy4G8DM232ED8ltQ23uu80HtC8cfiefaLijvbCvYRAcMCX22ycl_I0zGg4nUZ_tfY74ggsMcaec6Xny5NSCV38qnAfGx3XAE0hKaOqXOuZWd3Svqo2ST0iIKhxdMSL21l0Ruw6g93Hyurl6ihswQGq9A-5dfzZL7ceoR7P1tf0hs_lmMKKe_VPL5UqNpb23UjUHypHjYFz7nVR766VQ7RAa27TAhWSY_mUKcDF1k5Vz_idMmPHPAv-P9oVAoSHex1xM9s7DDJnMyJhqFaOctYElbKnUofpKyMh8PNjizrGduQAuPBfXnibxVn7gZeNi6UK7eISf-i1Zamz5bGKcYshXAm_bcWiZzdByV9Z8ERPF54QDhDVBvrFOMrbUEx-EB98Qo-C2X0pAtAHvkvcigysYRAjizSI84ZPk7zBTR2KARClnAm5e_XygijwyCirIfOnYrKZaDjQmOsaQFiP-R9a3wfLy0ZrzC4HDme_m6TMdznyFyPc4mrp4GbFNXhJI0fiycEt3azxmqnTGKlIuRzQMxW4FggPuWOhV2lB9GOEWG_Cik3qwdcpRQApoOcdsRfo5buUNHm_fnDUcDxzWghz5IEUvrJJCjwYlKLi-l3uoH-jg4UZPu1JgRwZ7wouK2-sqb5RIt_3RK3i_L6TTsYItXBd14Ds24L8YdBMc9nPbb4UuRpTWxc8H5BTPIZY8sMt-eb6nr9j7MnGXaFEhnwSNO0edrY6ZnwLZK4eFzrE_ijzsLPsAuSJGe0Zl_g3ydMZhr9gvcRTQM595VdzvTdDp8l333X20dU4pKZH1Qvy8zTCpdr6UmLHCaWidtqVWjcCoWvGeypQLmxHDQytQGwNs0msJ7tinYagrnJ3FKq7e5WUsDUwfES1hV0jsYzqT2vOOv3JuLhyqAMvv8awCPjO47EIsF8hw3xlScWZVRog38UPfUwaBNmQhMDj23KYKFtQVpsLTwUg8BBMXAa1n4dEKjwW9IjjYiAlSX8HvL_efsya1eJDFFNQr9_BD1fHCh6pwP